src/common/ceph_context.h

```diff
@@ -407,9 +407,16 @@ class CephContext {
     l_mempool_items,
     l_mempool_last
   };
+  enum {
```

Comment:
l_daemon_first = 1000000,
why is this a bare enum?  why that starting value?

Reply:
Yeah, I have the same question.

Reply:
That's how perfcounters are done elsewhere, I grepped through the source and looked for the current highest one so I would make sure I didn't overlap with it. (I meant to add 1000 and didn't, though, I'll fix that.)